app/views/stories/show.html.erb

```diff
@@ -157,23 +156,22 @@
           </ul>
         <% end %>
 
-        <h4><%= pluralize(@story.collaborators.count, "Contributor") %></h4>
+        <h4><%= pluralize(@collaborators.size, "Contributor") %></h4>
 
         <div class="members">
-          <% present_collection(@story.collaborators).each do |user| %>
+          <% present_collection(@collaborators).each do |user| %>
             <%= user.avatar(30) %>
           <% end %>
         </div>
 
-        <% if @story.tasks.any? %>
+        <div class="overall-task-progress">
```

Comment:
This is now dealt with entirely in JS as that logic needed to be there anyway so might as well save on ruby and just load static html for JS to deal with